local_planner/cfg/local_planner_node.cfg

```diff
@@ -30,26 +30,26 @@ gen.add("min_realsense_dist_", double_t, 0, "Discard points closer than that", 0
 gen.add("min_plane_percentage_", double_t, 0, "Discard planefits with less inliers", 0.7, 0, 1)
 gen.add("avoid_radius_", double_t, 0, "min dist before absolutely avoiding", 2.5, 0, 10)
 gen.add("min_dist_backoff_", double_t, 0, "min dist before backing off", 1.5, 0, 10)
-gen.add("pointcloud_timeout_hover_", double_t, 0, "after this timeout the controller sends hover commands", 0.25, 0, 10)
+gen.add("pointcloud_timeout_hover_", double_t, 0, "after this timeout the controller sends hover commands", 0.5, 0, 10)
 gen.add("pointcloud_timeout_land_", double_t, 0, "after this timeout the controller sends hover commands", 15, 0, 1000)
-gen.add("reproj_age_", int_t, 0, "maximum age of a reprojected point", 50, 0, 1000)
+gen.add("reproj_age_", int_t, 0, "maximum age of a reprojected point", 0, 0, 1000)
```

Comment:
Does this change aply to all configs? I.e. do we want to disable it for single cams as well?

Reply:
I don't know what the proper way of doing that in ROS would be. Your solution sounds fine.